Makefile

```diff
@@ -56,6 +61,11 @@ start-jenkins-build-webhook-proxy:
 	ocp-scripts/start-and-follow-build.sh --build-config jenkins-webhook-proxy
 .PHONY: start-jenkins-build-webhook-proxy
 
+# PROVISIONING APP
+## Install the gloabl provision app for the cluster
+install-provisioning-app:
+	cd provisioning-app/ocp-config && tailor apply --namespace ods --selector app=prov-app
```

Comment:
I think it would be safer if we move the selector into the `Tailorfile` - to avoid accidental damage.